src/schemaQuery.js

```diff
@@ -1,80 +1,85 @@
-var __awaiter =
-  (this && this.__awaiter) ||
-  function (thisArg, _arguments, P, generator) {
-    function adopt(value) {
-      return value instanceof P
-        ? value
-        : new P(function (resolve) {
-            resolve(value);
-          });
-    }
+"use strict";
```

Comment:
Do we need to upload Schema Query JS files to github? what are we preserving here?